engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/ReturningType.scala

```diff
@@ -33,3 +36,7 @@ trait ServiceReturningType {
   def returnType(parameters: Map[String, (TypingResult, Option[Any])]): TypingResult
 
 }
+
+case class CustomParameterValidationException(message: String, description: String, paramName: String) extends Exception(message)
+
+case class CustomServiceValidationException(message: String) extends Exception(message)
```

Comment:
can we rename it to CustomNodeValidationException?

Reply:
Done